Actions/AL-Go-Helper.ps1

```diff
@@ -416,13 +416,19 @@ function GetBcContainerHelperPath([string] $bcContainerHelperVersion) {
 #
 function DownloadAndImportBcContainerHelper([string] $baseFolder = $ENV:GITHUB_WORKSPACE) {
     $params = @{ "ExportTelemetryFunctions" = $true }
-    $repoSettingsPath = Join-Path $baseFolder $repoSettingsFile
 
     # Default BcContainerHelper Version is hardcoded in AL-Go-Helper (replaced during AL-Go deploy)
     $bcContainerHelperVersion = $defaultBcContainerHelperVersion
+
+    if ("$env:settings" -ne "") {
+        $repoSettingsPath = Join-Path ([System.IO.Path]::GetTempPath()) "$([Guid]::NewGuid().ToString()).json"
```

Comment:
Consider using $ENV:RUNNER_TEMP instead as it will automatically get cleaned by GitHub after the job is completed.